src/validation.cpp

```diff
@@ -2650,6 +2649,13 @@ bool CChainState::ConnectBlock(const CBlock& block, CValidationState& state, CBl
     const auto consensus = chainparams.GetConsensus();
     auto isEvmEnabledForBlock = IsEVMEnabled(pindex->nHeight, mnview, consensus);
 
+    uint64_t evmQueueId{};
```

Comment:
With queue creation moved to inside ConnectBlock from ConnectTip how do we delete the queue if ConnectBlock returns early with a failure? We may end up with dangling queues. cc/ @Bushstar

Reply:
Thanks. This was the reason for having the queue in ConnectTip, so if ConnectBlock returned early we could clean up the queue as construct block would be skipped at the end of ConnectBlock.

Reply:
Connect block changes are reverted commit 037d513